### PR TITLE
[ADD] fieldservice_partner_multi_relation

### PR DESCRIPTION
**Pull request by osi-scampbell**
Added fieldservice_res_parter_relation_type to connect fieldservice to partner_multi_relations

#88

### CODE REVIEW

**Review by smangukiya — APPROVED** [review]

File: fieldservice_partner_relation_type/README.rst [hunk 1]
```diff
@@ -0,0 +1,128 @@
+===========================
+Field Service - Maintenance
```
Comment (max3903) [inline fieldservice_partner_relation_type/README.rst]: Maintenance???

File: fieldservice_partner_relation_type/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+* Steven Campbell <smangukiya@opensourceintegrators.com>
```
Comment (max3903) [inline fieldservice_partner_relation_type/readme/CONTRIBUTORS.rst]: smangukiya?

**Review by max3903 — CHANGES_REQUESTED** [review]
@osi-scampbell Please rename the module as specified in the issue: fieldservice_partner_multi_relation

File: fieldservice_partner_relation_type/readme/CONTRIBUTORS.rst (hunk 2 above)
Comment (smangukiya) [reply fieldservice_partner_relation_type/readme/CONTRIBUTORS.rst]: @max3903 : Good catch, I have not noticed that :)

Comment (osi-scampbell) [reply fieldservice_partner_relation_type/readme/CONTRIBUTORS.rst]: Whoops! I used Sandip's README as a reference for mine

**Review by wolfhall — APPROVED** [review]

**Pull request merged**